geo/Geoserver.py

```diff
@@ -70,7 +90,7 @@ def get_manifest(self):
 
     def get_version(self):
         """
-        Returns the version of the geoserver.
+        Returns the version of the geoserver as JSON. It contains only the details of the high level components: GeoServer, GeoTools, and GeoWebCache
         """
         try:
             url = "{}/rest/about/version.json".format(self.service_url)
```

Comment:
Please remove the `rest` parameter from url. I think it is needed to check for all the requests.